.erb-lint.yml

```diff
@@ -0,0 +1,24 @@
+---
+EnableDefaultLinters: true
+linters:
+  Rubocop:
+    enabled: true
+    rubocop_config:
+      inherit_from:
+        - .rubocop.yml
+      Layout/ArgumentAlignment:
```

Comment:
Any particular reason for disabling these specifically?

Reply:
Yes, they don’t make sense in the context of ERB templates. Rubocop reads everything between the `<%` and the `%>` which are typically isolated from other code in the template.

For example, this code will throw errors:

```erb
<% subsection_status = subsection.status(@case_log) %>
```

* It is indented by 1 space
* It doesn’t have a new line after it
* It’s a useless assignment

None of which is helpful in this context.